draft-ietf-quic-transport.md

```diff
@@ -4584,13 +4584,14 @@ stateless_reset_token (0x0002):
   reset ({{stateless-reset}}) for the connection ID negotiated during the
   handshake.
 
-max_packet_size (0x0003):
+max_udp_size (0x0003):
```

Comment:
I think `max_udp_payload_size` is clearer than `max_udp_size` but this is still better than `max_packet_size`.

Reply:
I like short names but if this in the consensus I'll change it.

Reply:
OK, I'm going to take 3 thumbs up as "consensus" for this cosmetic issue, and will fix it in the next commit.